liferay-echo/liferay-portal
# LPS-192969 Add image size warning to ImageSelectorSize

Feature flag: `LPS-187285`

Steps to test:
1. Create a content page.
2. Add an image fragment.
3. "Direct select" an image that is larger than 500kb.
4. See the warning message.
5. Enable lazy loading or select some smaller image size to hide the warning.

> The warning text only appears in some use cases where we have the image size. We still need to investigate if we can include this warning in more situations.

![image](https://github.com/liferay-echo/liferay-portal/assets/800645/a3a0ad66-ff73-4b6d-b43f-df3a7323d13d)
